### DISCUSSION TITLE
Ctrl + C closes the pane

### DISCUSSION THREAD

**Discussion by indahud** [zellij-org/zellij#1665] [Q&A]
I am using bash. I am trying to run nodejs application. I want to stop the code running in my pane. When I enter `Ctrl+C` it closes the immedietly.  Is there any way to stop the running code?

**Comment by jaeheonji** [comment]
zellij doesn't bind `Ctrl+C` keys in default mode. In my case the interrupt works without any problem.
Can you check if another program is consuming the key? (ex. terminal emulator)
